src/app/modules/shared/models/project-type.model.ts

```diff
@@ -0,0 +1,6 @@
+export interface ProjectType {
+  id: string;
+  name: string;
+  description: string;
+  tenant_id?: string;
```

Comment:
@enriquezrene I'm not sure if tenant_id still should be a property. We can get it from session storage and in this case, we don't use it